src/serialize.h

```diff
@@ -152,12 +152,11 @@ inline float ser_uint32_to_float(uint32_t y)
 // i.e. anything that supports .read(char*, size_t) and .write(char*, size_t)
 //
 
-enum
+enum SerializeType
```

Comment:
That's not actually what the standard size. Enum is implementation defined,
but *no larger than* int. It could be char.

On Fri, Jul 10, 2015 at 8:57 AM, Pieter Wuille <notifications@github.com>
wrote:

> In src/serialize.h
> <https://github.com/bitcoin/bitcoin/pull/5448#discussion_r34368410>:
>
> > @@ -152,12 +152,11 @@ inline float ser_uint32_to_float(uint32_t y)
> >  // i.e. anything that supports .read(char*, size_t) and .write(char*, size_t)
> >  //
> >
> > -enum
> > +enum SerializeType
>
> However, an enum should always have type int, as long as the number of
> elements fits in an int, so I don't think that's necessary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/5448/files#r34368410>.
>